src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1176,6 +1175,8 @@ class DocBaseViewer extends BaseViewer {
             return;
         }
 
+        this.zoomControls.setCurrentScale(this.pdfViewer.currentScale);
```

Comment:
Should this go under the `if (!this.somePageRendered)` condition below?

Reply:
I don't think so because this isn't an initial render thing, it's everytime the pdf viewer re-renders which can result from initial render, zoom in/out, toggle fullscreen, etc...